e2e/specs/browser_config.js

```diff
@@ -0,0 +1,54 @@
+/**
+ * @license
+ * Copyright 2018-2021 Streamlit Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+describe("browser.config", () => {
+  before(() => {
+    cy.visit("http://localhost:3000/");
+  });
+
+  it("main menu is hidden", () => {
+    // click button to set hideMainMenu config option to true
```

Comment:
nit: I think the name of the button makes it clear enough what is happening here (and in the tests below) so that the comment is unnecessary